File: Selenium/Activities/Activity6.py
# Set up the Firefox Driver with WebDriverManger
from selenium.webdriver import ActionChains, Keys
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Set up the actions object
    actions = ActionChains(driver)
    # Open the browser to the URL
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("Title of the page is: ", driver.title)
    # Press the key of first letter of your name in caps
    actions.send_keys("A").pause(1).perform()
    # Press CTRL+a and the CTRL+c to copy all the text on the page
    actions.key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(Keys.CONTROL).perform()

File: Selenium/Activities/Activity6.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Action;
import org.openqa.selenium.interactions.Actions;

public class Activity6 {
    public static void main(String[] args) throws InterruptedException {
        // Setup the Firefox driver(GeckoDriver)
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        // Create a new instance of the Firefox driver
        WebDriver driver = new FirefoxDriver();
        // Initialize Actions object
        Actions builder = new Actions(driver);
        driver.get("https://v1.training-support.net/selenium/input-events");
        Thread.sleep(2000);
        System.out.println("Title of the page: " + driver.getTitle());
        // Press the key of first letter of your name in caps
        builder.sendKeys("A").pause(1000).build().perform();
        // Press CTRL+a and the CTRL+c to copy all the text on the page
        builder.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).build().perform();
        // Close the browser
        driver.quit();
    }
}
